lib/matplotlib/colorbar.py

```diff
@@ -478,6 +478,7 @@ def __init__(self, ax, mappable=None, *, cmap=None,
         self.ax.add_collection(self.dividers)
 
         self.locator = None
+        self.minorlocator = None
```

Comment:
What is this for?

Reply:
When we go through the new update path we try to update the minorlocator, but it complains about no attribute exists, so we should just make it available in __init__ right away.